packages/server/tests/clients/relayClient.ts

```diff
@@ -61,10 +64,11 @@ export default class RelayClient {
             this.logger.trace(`${requestIdPrefix} [POST] to relay '${methodName}' with params [${params}] returned ${JSON.stringify(res)}`);
             Assertions.expectedError();
         } catch (err) {
-            if (expectedRpcError.name == "Internal error"){
-                expectedRpcError = predefined.INTERNAL_ERROR(err.message);
-            }
-            Assertions.jsonRpcError(err, expectedRpcError);
+            // TODO: adapt to the new ethers v6 JsonRpcProvider error handling
```

Comment:
Usually not a fan of TODO's in code and also leaving these around when we can fix them now.
However, I do see this is a large PR so agree with fixing it in the next PR.
Let's have 1 ticket to capture all the TODO's of "adapting to the new ethers v6 JsonRpcProvider error handling"

Reply:
[The ticket.](https://github.com/hashgraph/hedera-json-rpc-relay/issues/1617)

Reply:
Okay, let's remove the comments then since we have the ticket.